vllm/config.py

```diff
@@ -1958,6 +1962,35 @@ class VllmConfig:
     decoding_config: Optional[DecodingConfig] = None
     observability_config: Optional[ObservabilityConfig] = None
     prompt_adapter_config: Optional[PromptAdapterConfig] = None
+    quant_config: Optional[QuantizationConfig] = field(init=False)
```

Comment:
Why not default to None like the other configs?

Reply:
good point, changed in [a4995fa](https://github.com/vllm-project/vllm/pull/9978/commits/a4995fa3ab7670d9032c373f53ed421922b4830e)